powerVCF.psm1

```diff
@@ -1806,3 +1806,104 @@ try {
 
     }	
 }
+
+######### Start CEIP Operations ##########
+
+Function Get-VCFSettingCEIP {
+
+<#
+    .SYNOPSIS
+    Retrieves the setting for CEIP of the connected SDDC Manager
+	
+    .DESCRIPTION
+    The Get-VCFSettingCEIP cmdlet Retrieves the setting for CEIP of the connected SDDC Manager. 
+	
+    .EXAMPLE
+    PS C:\> This example shows how to get the current setting of CEIP
+	
+	PS C:\> Get-VCFSettingCEIPs
+	
+#>
+
+    $headers = @{"Accept" = "application/json"}
+    $headers.Add("Authorization", "Basic $base64AuthInfo")
+    $uri = "https://$sddcManager/v1/system/ceip"
+    try { 	
+		    $response = Invoke-RestMethod -Method GET -URI $uri -headers $headers
+		    $response      
+        }
+        catch {
+            # Call the function ResponseExeception which handles execption messages
+            ResponseExeception
+        }
+
+}
+Export-ModuleMember -Function Get-VCFSettingCEIP
+
+Function Set-VCFSettingCEIP {
+
+<#
+    .SYNOPSIS
+    Sets the CEIP status (Enabled/Disabled) of the connected SDDC Manager
+	
+    .DESCRIPTION
+    The Set-VCFSettingCEIP cmdlet configuers the setting for CEIP of the connected SDDC Manager. 
+	
+    .EXAMPLE
+    PS C:\> This example shows how to disable CEIP of the connected SDDC Manager
+	
+    PS C:\> Set-VCFSettingCEIP ENABLE
```

Comment:
Do you not need to pass -ceipSetting? That would align it to other functions that accept multiple params

Reply:
Added to example